File: src/main/java/replication/model/Order.java
package replication.model;


import javax.persistence.Entity;
import javax.persistence.Id;
import java.time.LocalDate;

@Entity
public class Order {
    @Id
    private int id;
    private Book book;
    private Reader reader;
    private LocalDate createDate;
    private LocalDate execDate;
    private Library library;
    private String status;
}

File: src/main/webapp/my-system/src/app/data/Order.ts
import {Reader} from "./Reader";
import {Library} from "./Library";
import {OrdId} from "./OrdId";

export class Order {
  orderId: OrdId;
  reader: Reader;
  createDate: string;
  execDate: string;
  library: Library;
  status: string;
}
